en/InfoMotion/DataSourceSchema.md

```diff
@@ -9,11 +9,42 @@ The `timestamp` key must have a Unix Timestamp value.
 InfoMotion uses the `timestamp` for daterange picker, timeline and querying. 
 The `value` object supplies the InfoType with data to visualize. 
 
+##Sample Data 
```

Comment:
please put space between "##" and the title. Some markdown parser can't parse a sentence like this. https://gyazo.com/50967ef32a737d2ae3be3ff6d4c6c0bd